__tests__/src/components/WalletCard/WalletCard.js

```diff
@@ -8,7 +8,7 @@ import Images from '../../../../src/global/AssetsImages';
 describe('WalletCard component tests', () => {
   test('Rendering', () => {
     const tree = renderer.create(<WalletCard
-      imagePath={Images.ethereumLogo}
```

Comment:
hi, we should definitely keep the ethereum logo for ethereum.

Reply:
ok, will change it